css/src/components/accordion.scss

```diff
@@ -0,0 +1,156 @@
+/* stylelint-disable selector-no-qualifying-type */
+$accordion-font-size-sm: $font-size-8 !default;
+$accordion-font-size-md: $font-size-7 !default;
+$accordion-font-size-lg: $font-size-6 !default;
+
+$accordion-spacing-xs: $spacer-2 !default;
+$accordion-spacing-sm: $spacer-3 !default;
+$accordion-spacing-md: $spacer-4 !default;
+$accordion-spacing-lg: $spacer-5 !default;
+$accordion-spacing-offset-sm: $spacer-6 !default;
+$accordion-spacing-offset-md: $spacer-7 !default;
+$accordion-spacing-offset-lg: $spacer-8 !default;
+
+$accordion-transition: transform 0.15s !default;
+
+// Spacing and font-size
+
+@mixin accordion-summary-sizes($font-size, $padding-size, $icon-offset-size) {
+	$bg-icon-offset-size: calc($icon-offset-size + 0.25rem);
+
+	.accordion-content,
+	summary {
+		padding-block: $padding-size;
+	}
+
+	summary {
+		font-size: $font-size;
+
+		&:not(.accordion-icon-end) {
+			padding-inline-start: $icon-offset-size;
+
+			&::before {
+				inset-block-start: calc($control-padding-horizontal + $padding-size);
+			}
+		}
+
+		&.accordion-icon-end {
+			padding-inline-end: $icon-offset-size;
+
+			&::after {
+				inset-block-start: calc($padding-size * 1.5);
+			}
+		}
+	}
+
+	.accordion-content {
+		padding-block: $padding-size;
```

Comment:
Seems like line 23 already sets the same padding for `.accordion-content`?

Reply:
This was removed.